### PR TITLE
setup: Create bin scripts with entry_points

### LINKED ISSUE #122: Startup script assumes `python` is python3
The startup script (`/usr/bin/hotdoc`) should probably explicitly have `python3` in its shebang line. I'm trying to run it from within the GNOME flatpak SDK where `python` is python2, so it fails at `from hotdoc.run_hotdoc import ...`.

### PR DESCRIPTION
**Pull request by ptomato**
Instead of supplying our own scripts that get installed into $prefix/bin
and having to deal with different shebang lines and encodings, use
setuptools' mechanism for autogenerating a script given an entry point.

Closes #122.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -352,7 +350,10 @@ def run_tests(self):
         extras_require=EXTRAS_REQUIRE,
         entry_points={
             'hotdoc.extensions': ('get_extension_classes = '
-                                  'hotdoc.extensions:get_extension_classes')},
+                                  'hotdoc.extensions:get_extension_classes'),
+            'console_scripts': [
+                'hotdoc=hotdoc.run_hotdoc:_main',
```
Comment (MathieuDuponchelle) [inline setup.py]: Any reason for having the underscore prefix to main here but not in the other script? Just curious, I don't really mind either way :)

Comment (ptomato) [reply setup.py]: There was already an underscore on `_main` in that file, so I kept it the way it was. I can remove it for consistency if you prefer.

**Comment by ptomato** [comment]
I removed the underscore from main, so it's consistent now.

**Comment by ptomato** [comment]
Ping?

**Comment by MathieuDuponchelle** [comment]
pong!

**Pull request merged**